File: QViz.NET/Periscope.QViz/JSON/Count.cs
namespace Periscope.QViz.JSON
{
	/// <summary>
	/// JSON Model for Test Case Counts
	/// </summary>
	public class Count
	{
		/// <summary>
		/// Type of the Test Case
		/// </summary>
		public string testCaseType;
		
		/// <summary>
		/// Total number of all Test Cases
		/// </summary>
		public int totalTestCount;
		
		/// <summary>
		/// Total number of automated Test Cases only
		/// </summary>
		public int automatedTestCount;

		/// <summary>
		/// Default Constructor
		/// </summary>
		public Count()
		{
			this.testCaseType = "";
			this.totalTestCount = 0;
			this.automatedTestCount = 0;
		}
	}
}

File: QViz.Java/src/main/java/com/periscope/qviz/json/Count.java
package com.periscope.qviz.json;

/**
 * JSON Model for Test Case Counts
 */
public class Count {

	/**
	 * Type of the Test Case
	 */
	public String testCaseType;

	/**
	 * Total number of all Test Cases
	 */
	public Integer totalTestCount;

	/**
	 * Total number of automated Test Cases only
	 */
	public Integer automatedTestCount;

	/**
	 * Default Constructor
	 */
	public Count() {
		this.testCaseType = "";
		this.totalTestCount = 0;
		this.automatedTestCount = 0;
	}

}
